New ALB Module

luizfdsilva opened the pull request.

luizfds committed:
added files

christina-roperto reviewed (commented):
code looks good, but I ran into 

```
Error: Error creating Security Group: InvalidGroup.Duplicate: The security group 'tf-ecs-sg' already exists for VPC 'vpc-00271cb0ff3152133'
	status code: 400, request id: 5824d7f3-05cd-4d74-bf59-bb614d070233
```
during `make deploy`.

I will investigate a bit and see if its something on my side.

christina-roperto commented:
The duplicated sg was my bad. Sorry! 

But now I'm getting 
```
Error: InvalidParameterException: You cannot specify an IAM role for services that require a service linked role. "wdpress-dev"
```
I'll investigate it

luizfds committed:
Removed iam_role from ecs service and changed alb port to 80

luizfdsilva commented:
> The duplicated sg was my bad. Sorry!
> 
> But now I'm getting
> 
> ```
> Error: InvalidParameterException: You cannot specify an IAM role for services that require a service linked role. "wdpress-dev"
> ```
> 
> I'll investigate it

Thanks Christina, I removed iam_role from ecs service and changed alb port to 80.
